src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsworkload/jpa/entity/EventManagerAuditEntity.kt

```diff
@@ -0,0 +1,41 @@
+package uk.gov.justice.digital.hmpps.hmppsworkload.jpa.entity
+
+import org.springframework.data.annotation.CreatedDate
+import org.springframework.data.jpa.domain.support.AuditingEntityListener
+import java.time.ZonedDateTime
+import javax.persistence.Column
+import javax.persistence.Entity
+import javax.persistence.EntityListeners
+import javax.persistence.GeneratedValue
+import javax.persistence.GenerationType
+import javax.persistence.Id
+import javax.persistence.JoinColumn
+import javax.persistence.ManyToOne
+import javax.persistence.Table
+
+@Entity
+@Table(name = "EVENT_MANAGER_AUDIT")
+@EntityListeners(AuditingEntityListener::class)
+data class EventManagerAuditEntity(
+  @Id
+  @Column
+  @GeneratedValue(strategy = GenerationType.IDENTITY)
+  val id: Long? = null,
+
+  @Column
+  val justificationNotes: String,
```

Comment:
Any reason for this field not being the same as is in the request, i.e. `evidenceContent` and `evidenceContent Sensitive`?

Reply:
Oh ok .. I don't think Pauline expected that terminology to be used verbatim but we ought to have a consistent way of referring to these.. and you are right we should have designed the json from this ticket.

Also, if we are going to do what is in the ticket then it would be `allocationJustificationNotes`... but i just prefer consistency